plugins/com.google.cloud.tools.eclipse.usagetracker/src/com/google/cloud/tools/eclipse/usagetracker/AnalyticsPingManager.java

```diff
@@ -55,6 +55,13 @@
   private static final Logger logger = Logger.getLogger(AnalyticsPingManager.class.getName());
 
   private static final String ANALYTICS_COLLECTION_URL = "https://ssl.google-analytics.com/collect";
+  private static final String CLEARCUT_COLLECTION_URL = " https://play.google.com/log";
+  
+  // for Concord
```

Comment:
"// for Concord Clearcut"

since this is specially for Clearcut.

Reply:
Should this be in `CloudToolsInfo`? It has `METRICS_NAME`:

```java
  // Don't change the value; this name is used as an originating "application" of usage metrics.
  public static final String METRICS_NAME = "gcloud-eclipse-tools";
```

Reply:
done